tests/baselines/reference/quickInfoJsDocTags13.baseline

```diff
@@ -0,0 +1,353 @@
+=== /tests/cases/fourslash/quickInfoJsDocTags13.ts ===
+// /**
+//  * @param a - a param
+//  * @param bindingelement1 - for { b, c }
+//  * @param b - b param
+//  * @param bindingelement2 - for { e }
+//  * @param c - c param
+//  * @param d - d param
+//  */
+// function foo(a: string, { b, c }: { b: string, c: string }, d: string, { e }: { e: string }) {
+//     a
+//     ^
+// | ----------------------------------------------------------------------
+// | (parameter) a: string
+// | - a param
+// | @param a - a param
```

Comment:
Why does the quickinfo for normal parameters has an extra line with the original jsdoc, e.g. `@param a - a param`, and the binding element params don't?